### DISCUSSION TITLE
ingest_folder.py throws error

### DISCUSSION THREAD

**Discussion by bspidel** [imartinez/privateGPT#1552] [Q&A]
I have commit 3d301d0c6f4c177ec3fba55867a07b4c67e19d97
Running on hp i3 3.7gh with 24g of ram Windows 11 with WSL Ubuntu 22.04.3
I can run server and client. I can ingest individual files using the client. I wish to ingest many hundreds of files.

when I run from privateGPT folder: 

   make ingest /mnt/c/SpidelRoot/Projects/vs-dcs/resources/obs

I expect hundreds of small files to be ingested.
What I get is

   Traceback (most recent call last):
  File "/mnt/c/SpidelRoot/Projects/privateGPT/scripts/ingest_folder.py", line 7, in <module>
    from private_gpt.di import global_injector
ModuleNotFoundError: No module named 'private_gpt'
make: *** [Makefile:52: ingest] Error 1

I am not that skilled with python so I don't know how to give it what it wants.

Please assist.

**Comment by mamema** [comment]
have the same error still today
